src/Microsoft.AspNetCore.Mvc.Core/Internal/ApiBehaviorOptionsSetup.cs

```diff
@@ -29,15 +29,14 @@ public void Configure(ApiBehaviorOptions options)
             IActionResult GetInvalidModelStateResponse(ActionContext context)
             {
                 var errorDetails = _errorDescriptionFactory.CreateErrorDescription(
-                    context.ActionDescriptor, 
+                    context.ActionDescriptor,
                     context.ModelState);
 
                 var result = (errorDetails is ModelStateDictionary modelState) ?
                     new BadRequestObjectResult(modelState) :
                     new BadRequestObjectResult(errorDetails);
 
                 result.ContentTypes.Add("application/json");
-                result.ContentTypes.Add("application/json");
```

Comment:
Think this was meant to be `application/xml`

Reply:
Okay. I didn't expect that